pkg/controllers/updaterun/initialization_integration_test.go

```diff
@@ -1065,6 +1066,10 @@ func generateSucceededInitializationStatusForSmallClusters(
 			generateTrueCondition(updateRun, placementv1beta1.StagedUpdateRunConditionInitialized),
 		},
 	}
+	for i := range numUnscheduledClusters {
+		status.DeletionStageStatus.Clusters = append(status.DeletionStageStatus.Clusters,
+			placementv1beta1.ClusterUpdatingStatus{ClusterName: fmt.Sprintf("unscheduled-cluster-%d", i)})
```

Comment:
Can we create a format specifier var for unscheduled-cluster-%d and use it everywhere